master/reference/calicoctl/resources/hostendpoint.md

```diff
@@ -3,14 +3,27 @@ title: Host Endpoint Resource (HostEndpoint)
 canonical_url: 'https://docs.projectcalico.org/v3.2/reference/calicoctl/resources/hostendpoint'
 ---
 
-A host endpoint resource (`HostEndpoint`) represents an interface attached to a host that is running {{site.prodname}}.
+Host endpoint resources (`HostEndpoint`) come in two flavors.
```

Comment:
I'm not sure if this is the right characterization of host endpoitns, or at least not how I was imagining we'd represent it.

Rather than say there are two flavors, I feel it might be simpler / easier to understand if we say something like this:
- "A host endpoint resource represents one or more real or virtual interfaces attached to a host that is running Calico"

Then, we talk about how the various ways you can specify which interfaces it represents:
- By declaring specific IP addresses.
- By declaring a specific interface name.
- By selecting all interfaces on the host.

WDYT? The current wording doesn't seem wrong, but perhaps is harder to reason about, and loses the high-level "What does a host endpoint represent?" messaging.

Reply:
OK, fine, let me see if I can rework along those lines...